nixos/modules/services/network-filesystems/netatalk.nix

```diff
@@ -135,10 +80,11 @@ in
         Type = "forking";
         GuessMainPID = "no";
         PIDFile = "/run/lock/netatalk";
-        ExecStartPre = "${pkgs.coreutils}/bin/mkdir -m 0755 -p /var/lib/netatalk/CNID";
-        ExecStart  = "${pkgs.netatalk}/sbin/netatalk -F ${afpConfFile}";
+        ExecStartPre =
+          "${pkgs.coreutils}/bin/mkdir -m 0755 -p /var/lib/netatalk/CNID";
```

Comment:
Not related to this PR, but is there any reason we couldn't replace this with `StateDirectory`? If so... any motivation to do that in this PR? Or some other time.

Reply:
Done.

Reply:
Sorry, what I meant is that `StateDirectory = [ "netatalk/CNID" ];` should entirely replace the need for an `ExecStartPre` and `mkdir`.

Reply:
Ok, good. Done.

Reply:
Awesome! Thanks @ehmry :tada: